src/charm.py

```diff
@@ -152,12 +141,16 @@ def _configure(self, _):
                 logger.error(f"Failed to replan; pebble plan: {container.get_plan().to_dict()}")
                 raise e
 
-        if (
```

Comment:
By removing these lines and not adding anything equivalent you are changing the behaviour of this charm. Actually you are reverting the behaviour to what existed before these lines were added. So I think it may be important to understand why these lines were added and why it is ok to now remove them. This is not entirely clear to me.